meson.build

```diff
@@ -479,7 +479,6 @@ srcfiles_unittest = files(
     'test/redfish-core/lib/log_services_dump_test.cpp',
     'test/redfish-core/lib/manager_diagnostic_data_test.cpp',
     'test/redfish-core/lib/metadata_test.cpp',
-    'test/redfish-core/lib/power_subsystem_test.cpp',
```

Comment:
Why are we removing this?

Reply:
Original change was done by Shawn who said I think it was removed because it was failing CI downstream.  Not 100% sure, but that is my recollection.

Reply:
When I did this back in 1050, I had originally commented out the test because I believe it was failing CI.  Gunnar suggested removing the file instead.

https://github.com/ibm-openbmc/bmcweb/pull/654

If I remember correctly, CI wasn't failing because my change was incorrect.  It was more because the test case didn't set up a real environment, and then things failed when it tried to do real things.

Reply:
I don't see what in the test would fail. Can we try with this test in ?